spyder/app/mainwindow.py

```diff
@@ -1466,7 +1466,7 @@ def update_lsp_logs(self):
     def post_visible_setup(self):
         """Actions to be performed only after the main window's `show` method
         was triggered"""
-        for _plugin_id, plugin in self._PLUGINS.items():
+        for __, plugin in self._PLUGINS.items():

```

Comment:
`_plugin_id` is more readable than `__`

Reply:
Thanks @goanpeca for the suggestion, but we're adopting the convention of using `__` for unnecessary variables in loops. Using a name for those variables, even if it starts with an underscore, gives the impression that it's required and somehow missing in the code below.

Actually, that was a small source of confusion for @dalthviz and I when we were reviewing this PR, so I suggested him to do this change.